app/MMB/src/scenes/tripServices/insurance/MoreInfoScene.js

```diff
@@ -0,0 +1,374 @@
+        key="interpretation_and_translation-1-V"
+        code="V"
+      />,
+      <TextIcon
+        style={styles.icon}
+        key="interpretation_and_translation-2-V"
+        code="V"
+      />,
+    ],
+    [
+      <Translation
+        key="booking.passenger.insurance.delayed_baggage_coverage"
+        id="mmb.trip_services.insurance.selection.more_info.delayed_baggage_coverage"
+      />,
+      <Translation key="1-—" passThrough="—" />,
+      <Translation key="2-100 €" passThrough="100 €" />,
+    ],
+    [
+      <Translation
+        key="booking.passenger.insurance.flight_delays_or_cancellations_coverage"
+        id="mmb.trip_services.insurance.selection.more_info.flight_delays_or_cancellations_coverage"
+      />,
+      <Translation
+        key="flight_delays_or_cancellations_coverage-1-—"
+        passThrough="—"
+      />,
+      <Translation
+        key="flight_delays_or_cancellations_coverage-2-20 €/per hour, max. 100 €"
+        passThrough="20 €/per hour, max. 100 €"
+      />,
+    ],
+    [
+      <Translation
+        key="booking.passenger.insurance.missed_flight_coverage"
+        id="mmb.trip_services.insurance.selection.more_info.missed_flight_coverage"
+      />,
+      <Translation key="missed_flight_coverage-1-—" passThrough="—" />,
+      <Translation key="missed_flight_coverage-2-100 €" passThrough="100 €" />,
+    ],
+    [
+      <Translation
+        key="booking.passenger.insurance.flight_cancellation"
+        id="mmb.trip_services.insurance.selection.more_info.flight_cancellation"
+      />,
+      <Translation
+        key="flight_cancellation-1-booking.passenger.insurance.to_amount_paid_for_flights_and"
+        id="mmb.trip_services.insurance.selection.more_info.to_amount_paid_for_flights_and"
+      />,
+      <Translation
+        key="flight_cancellation-2-booking.passenger.insurance.to_amount_paid_for_flights_and"
+        id="mmb.trip_services.insurance.selection.more_info.to_amount_paid_for_flights_and"
+      />,
+    ],
+    [
+      <Translation
+        key="booking.passenger.insurance.co_insurance_causes_flights_cancellation"
+        id="mmb.trip_services.insurance.selection.more_info.co_insurance_causes_flights_cancellation"
+      />,
+      <Translation
+        key="co_insurance_causes_flights_cancellation-1-10%"
+        passThrough="10%"
+      />,
+      <Translation
+        key="co_insurance_causes_flights_cancellation-2-10%"
+        passThrough="10%"
+      />,
+    ],
+    [
+      <Translation
+        key="booking.passenger.insurance.co_insurance_causes_subsequent_services"
+        id="mmb.trip_services.insurance.selection.more_info.co_insurance_causes_subsequent_services"
+      />,
+      <Translation
+        key="co_insurance_causes_subsequent_services-1-20%"
+        passThrough="20%"
+      />,
+      <Translation
+        key="co_insurance_causes_subsequent_services-2-20%"
+        passThrough="20%"
+      />,
+    ],
+  ];
+
+  function onPress() {
```

Comment:
this is just really a hack for not triggering the eslint rule that does not allow arrow functions in jsx. This function will also be re created on each re render

Please rather create a class.